src/shared/lib/OncoKbUtils.ts

```diff
@@ -80,6 +80,28 @@ const LEVELS = {
     all: ['4', 'R3', '3B', '3A', 'R2', '2B', '2A', 'R1', '1', '0']
 };
 
+export const ALTERATIONS = {
+    mutation: "MUTATION",
+};
+
+/**
+ * Generates <query id, indicator> map.
```

Comment:
Actually, this is a manual comment. Right, it seems a bit redundant, since we already have a `@returns` with the proper `TypeScript` definition.